polaris-migrator/src/utilities/sass.ts

```diff
@@ -132,6 +132,26 @@ export function isTransformableLength(
   return transformableLengthUnits.includes(dimension.unit);
 }
 
+/**
+ * All transformable duration units. These values are used to determine
+ * if a decl.value can be converted to pixels and mapped to a Polaris custom property.
+ *
+ * Note: <time> is a dimension with 's' or 'ms' as the unit:
+ * https://w3c.github.io/csswg-drafts/css-values-3/#time-value
+ */
+export const transformableDurationUnits = ['s', 'ms'];
+
+export function isTransformableDuration(
+  dimension: false | Dimension,
+): dimension is Dimension {
+  if (!dimension) return false;
+
+  // Zero is the only unitless dimension our duration transforms support
+  if (isUnitlessZero(dimension)) return true;
```

Comment:
Oooh, I didn't know about this utility, thanks!

Interestingly, it doesn't convert the string to a number for comparison. I'm not sure what the CSS spec says, but conceivably this could miss a couple cases:

- `+0`
- `0.0`

To handle those, I used a type cooercion: `Number(unit.number) === 0`

Is it worth adding this to the `isUnitlessZero` function?